### ISSUE TITLE
Admin configuration cannot be generated for simple AppBundle

### ISSUE THREAD

**Issue by peric** [symfony2admingenerator/GeneratorBundle#88]
After running ```app/console sensio/generator-bundle``` and trying to generate configuration for an Entity inside simple bundle (AppBundle), you get an error message.

```
[InvalidArgumentException]
  The namespace must contain a vendor namespace (e.g. "VendorName\AppBundle" instead of simply "AppBundle").
  If you've specified a vendor namespace, did you forget to surround it with quotes (init:bundle "Acme\BlogBundle")?
```

This happens because of 
https://github.com/sensiolabs/SensioGeneratorBundle/issues/290

Pull request for that was accepted but it's still in master.

After it's released and after the update on this bundle, some things will also need adapted here. For example, I've tried to use master version of SensioGeneratorBundle and have received message below.

```
[Symfony\Component\Debug\Exception\ContextErrorException]
  Runtime Notice: Declaration of Admingenerator\GeneratorBundle\Command\GenerateAdminAdminCommand::updateRouting() should be compatible with Sensio\Bundle\GeneratorBundle\Command\GenerateBundleCommand::updateRouting(Symfony\Component\Console\Output\OutputInterface $output, Sensio\Bundle\GeneratorBundle\Model\Bundle $bundle)
```

**Comment by sescandell** [comment]
Hi @peric 

Thanks for your report.

**Comment by ksn135** [comment]
Fixed in my PR #149

**Comment by sescandell** [comment]
Closing as this has been fixed thanks to @ksn135
